third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -612,7 +621,12 @@ where
                     // Resolve the function. This may lead to loading the code related
                     // to this function.
                     let callee = lazy_function
-                        .with_resolved_function(module_storage, |f| Ok(f.clone()))
```

Comment:
action was there to clone it? what am I missing?

Reply:
yes it's a lot better now )

Reply:
I figured, there was another action as well, but overall quite obscure to generalize that as an action.